docs/gambit/index.md

```diff
@@ -10,7 +10,7 @@ Mutation testing is a technique for evaluating and improving
 The key idea is to introduce faults, called mutants, to the program under test
   and measure a test suite's ability to detect these mutants.
```

Comment:
"to detect the violations caused by these mutants"

to just detect run diff

Reply:
Changed to:
"The objective is to assess how well a test suite can identify and detect faulty logic in these mutants."